.rubocop.yml

```diff
@@ -13,6 +14,7 @@ AllCops:
 # Metrics/LineLength:
 #   Max: 120
 
+
 Style/StringLiterals:
   Enabled: true
   EnforcedStyle: single_quotes
```

Comment:
we don't have a consistent practice in DLSS, and we don't have an agreed upon practice in this spotlight sprint series yet, either.   If it's about shutting up hound, then add a comment to that effect.